.tests/test_gizmo.py

```diff
@@ -43,3 +43,10 @@ def test_ex6(capsys):
     captured = capsys.readouterr()
     assert captured.out.strip() == "F.r.a.n.k.f.u.r.t.e.r"
 
+def test_ex7():
+    """Ex7: return a list of relative paths given a list of subject identifiers"""
+    import gizmo
+    subjects = ['subj1', 'subj2']
+    flist = gizmo.relative_path(subjects)
+    assert flist[0]=="./subjects/mock_recording_subj1.rec"
+    assert flist[1]=="./subjects/mock_recording_subj2.rec"
```

Comment:
```suggestion
    assert flist[0] == "./subjects/mock_recording_subj1.rec"
    assert flist[1] == "./subjects/mock_recording_subj2.rec"
```
(let's follow PEP8 Python style guidelines)